membrane_curvature/base.py

```diff
@@ -110,9 +109,12 @@ class MembraneCurvature(AnalysisBase):
         gaussian_curvature = mc.results.average_gaussian_curvature
 
     The respective 2D curvature plots can be obtained using the `matplotlib`
-    package for data visualization via `imshow`. We recommend using the
-    `gaussian` interpolation.
+    package for data visualization via `contourf` or `imshow`.
```

Comment:
I would link to the matplotlib functions with intersphinx
```suggestion
    package for data visualization via :func:`~matplotlib.pyplot.contourf` or :func:`matplotlib.pyplot.`imshow`.
```
You need to check how exactly the intersphinx links need to be structured because I am guessing here.

You also need to add the intersphinx linking to your `conf.py`.

Reply:
Thanks, intersphinx was already there. Added the links to `contourf`, `imshow`, and while I was there also the one for `MDAnalysis.transformations.wrap.wrap`.